sematic/api_client.py

```diff
@@ -625,8 +627,12 @@ def _raise_for_response(
     raise exception
 
 
-def _url(endpoint) -> str:
-    return "{}{}".format(get_config().api_url, endpoint)
+def _url(endpoint: str) -> str:
```

Comment:
It might be cleaner to do it like this:
```suggestion
def _url(endpoint: str, uses_sockets: bool=False) -> str:
```

And then just switch which base url you use in this method. That way you don't have to replicate socket-io url construction everywhere it is used.

Reply:
This would need to be carried through `_request` as well, and it would make `_request` aware of this socket io concern.
If we want to have this solution be a temporary one that will be replaced with a more robust implementation, then we would want to have this change be as least intrusive as possible in order to be easily removable.

Reply:
Another option would be to have `_url` explicitly check whether `endpoint` contains certain paths that we know are related to sockets (`/api/v1/events`), and have it switch which base url it uses based on that. It just feels super weird to me that when you do "normal" stuff the URL is constructed for you, but when you do socket stuff, you have to construct the URL yourself each time.

Reply:
Done.